stdlib/public/core/StringBridge.swift

```diff
@@ -537,6 +596,213 @@ internal func _NSStringFromUTF8(_ s: UnsafePointer<UInt8>, _ len: Int)
   )._bridgeToObjectiveCImpl()
 }
 
+@_fixed_layout
+@usableFromInline
+@objc internal final class _SwiftNSMutableString :
+  _SwiftNativeNSMutableString, _AbstractStringStorage {
+  private var _contents: String
+  
+  internal init(_ str: String) {
+    _contents = str
+    super.init()
+    assert(_contents._guts.isFastUTF8)
+  }
+  
+  final internal var asString: String {
+    @_effects(readonly) @inline(__always) get {
```

Comment:
These can probably be moved above if you remove the `get`

Reply:
Unfortunately the inline can be but the _effects can't be :(